TFE/YBU-58/ybu-58.md

```diff
@@ -0,0 +1,73 @@
+# Run a Workload on a Multi-node Universe
+
+## Introduction
+
+In this hands-on lab, you will run a PostgreSQL workload on a multi-node Yugabyte Universe. In this example, you will use a three node cluster in a multi-zone topology that was set up in a previous lab. Each node will reside in the same region, but in a different availability zone. The purpose is to validate the Yugabyte Universe is functioning properly as well as examine the metric tools on the Yugabyte Platform.
+
+## Prerequisites
+
+* Verify the Universe is running
+
+* The `.pem` key file to connect the Platform server.
+
+## Verify the Universe is Operational
+
+Verify that the Universe is running by navigating to the Yugabyte Admin Console and selecting the Universe that will be running the sample workload. You can get to the Yugabyte Admin Console from the browser by navigating to the public IP of the [EC2 instance](https://us-west-2.console.aws.amazon.com/ec2/v2/home?region=us-west-2#Instances:instanceState=running) that hosts Platform management component.
+
+Select the Universe that will be running the workload. There will be a green "Ready" displayed next to the Universe name at the top of the page if it is operational. Also verify that the Primary Cluster has 3 nodes whose status is "Live".
```

Comment:
Add the step to retrieve the workload command prompt. This will reduce context swtiching